src/main/kotlin/com/terraformation/backend/tracking/PlantingSiteService.kt

```diff
@@ -26,4 +29,18 @@ class PlantingSiteService(
               PlantingSiteTimeZoneChangedEvent(site, event.oldTimeZone, event.newTimeZone))
         }
   }
+
+  @EventListener
+  fun on(event: PlantingSiteTimeZoneChangedEvent) {
+    val site = event.plantingSite
+
+    if (site.plantingSeasons.isNotEmpty()) {
+      plantingSiteStore.updatePlantingSeasons(
+          site.id,
+          site.plantingSeasons.map { UpdatedPlantingSeasonModel(it) },
+          event.newTimeZone ?: ZoneOffset.UTC,
```

Comment:
default to org's time zone?

Reply:
`newTimeZone` in the event already defaults to the org's time zone if there is one.